client/packages_test.go

```diff
@@ -87,3 +89,45 @@ func (cs *clientSuite) TestClientPackages(c *check.C) {
 		},
 	})
 }
+
+const (
+	appID = "chatroom.ogra"
+)
+
+func (cs *clientSuite) TestClientPackageCallsEndpoint(c *check.C) {
+	_, _ = cs.cli.Package(appID)
```

Comment:
Tests should always check the error result. When things go wrong, this is usually the most interesting piece of information we have to start debugging the issue.